programming/editors/vscode.md

```diff
@@ -35,3 +35,57 @@ you'll need to make the library available within the Python environment that you
 3. Run `pip install sr.robot3`.
 
 [code-completion]: https://en.wikipedia.org/wiki/Autocomplete#In_source_code_editors
+
+## Remote Debugging
+
+<div class="info">
+This documentation refers to a feature which is only available from software version `2023.1.0` and later.
```

Comment:
```suggestion
This documentation refers to a feature which is only available from software version <code>2023.1.0</code> and later.
```

Backticks don't render inside HTML

Reply:
Good spot. An alternative is to add `markdown="1"` to the containing HTML element.